modules/virt-configuring-vm-disk-sharing.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-disk-sharing{context}"]
+= Configuring disk sharing by using virtual machine disks
+
+You can configure block volumes so that multiple virtual machines (VMs) can share storage.
+
+The application running in the guest determines the storage option you must configure for the VM. A disk of type `disk` exposes the volume as an ordinary disk to the VM.
+
+.Prerequisites
+
+* The volume access mode must be `ReadWriteMany` (RWX) if the VMs that are sharing disks are running on different nodes.
++
+If the VMs that are sharing disks are running on the same node, `ReadWriteOnce` (RWO) volume access mode is sufficient.
+
+* The storage provider must support the required Container Storage Interface (CSI) driver.
+
+.Procedure
+
+. Create the `VirtualMachine` manifest for your VM to set the required values, as shown in the following example:
+
++
+[source,yaml]
+====
+----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: <vm_name>
+spec:
+ volumemode: Block
+ template:
+    spec:
```

Comment:
Bad indent

Reply:
Audrey, you can also check YAML examples like this [one](https://github.com/kubevirt/kubevirt/blob/main/examples/vmi-nocloud.yaml)

Reply:
Audrey, here is an example yaml with [spec.template.spec.domain.devices.disks](https://raw.githubusercontent.com/kubevirt/kubevirt/main/examples/vm-alpine-datavolume.yaml). Look at the indentation.

Reply:
Thanks, @apinnick